react-app/package.json

```diff
@@ -92,5 +95,8 @@
   "peerDependencies": {
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
+  },
+  "volta": {
+    "node": "18.0.0"
```

Comment:
What is this used for?

Reply:
Not sure why but when I switch node version through nvm it doesn't switch to latest and hence had to switch using volta(https://docs.volta.sh/guide/#why-volta) for switching to 18.0 version but have removed it now.